appliance/postgresql/Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM ubuntu-debootstrap:14.04
 
 ENV DEBIAN_FRONTEND noninteractive
 
+# Try to use the Hosts squid-deb-proxy and fall back if it's unreachable
+RUN (echo 'Acquire { Retries "0"; HTTP { Proxy "http://172.17.42.1:8000"; }; };' > /etc/apt/apt.conf.d/10_docker_hostproxy && apt-get update -qq 2>&1 | grep -q 'Unable to connect to 172.17.42.1:8000' && rm /etc/apt/apt.conf.d/10_docker_hostproxy) || true
```

Comment:
This should be combined with the `RUN` below to avoid creating another layer, and the config file should be removed at the end of the run so that it doesn't end up in the final image.